lib/runner.js

```diff
@@ -118,3 +118,8 @@ Runner.prototype.run = function () {
 			stats.passCount = stats.testCount - stats.failCount;
 		});
 };
+
+// Set custom assert module
+Runner.prototype.setAssertModule = function (assertModule) {
+	Test._setAssertModule(assertModule);
```

Comment:
Oops, I missed that there are `Runner` and `Test` there, sorry. I would leave `setAssertModule` on `Runner` and when you create a new instance of `Test`, assign it there - https://github.com/sindresorhus/ava/blob/master/lib/runner.js#L34.